include/zenoh-pico/api/primitives.h

```diff
@@ -2056,6 +2056,11 @@ z_result_t z_declare_background_subscriber(const z_loaned_session_t *zs, const z
 const z_loaned_keyexpr_t *z_subscriber_keyexpr(const z_loaned_subscriber_t *subscriber);
 #endif
 
+#if Z_FEATURE_BATCHING == 1
+z_result_t zp_batching_start(const z_loaned_session_t *zs);
+z_result_t zp_batching_stop(const z_loaned_session_t *zs);
```

Comment:
I don't think that `stop` good word for it, we should choose something which will show that this method is sending data.

Reply:
Maybe something like 
```
zp_batch_start
zp_batch_flush
```